playbooks/seapath_setup_main.yaml

```diff
@@ -9,23 +9,21 @@
 # This playbooks can also configure VMs created with build_debian_iso
 # This playbook also configures Yocto standalone machine (no cluster)
 
-- name: Gather ansible_distribution
+- name: Detect Seapath distribution
   hosts:
     - cluster_machines
     - standalone_machine
-  tasks:
-    - name: Gather only the ansible_distribution fact
-      ansible.builtin.setup:
-        filter: ansible_distribution
+  roles:
+    - detect_seapath_distro
 
 - import_playbook: seapath_setup_prerequiscentos.yaml
-  when: ansible_distribution | regex_search("CentOS|RedHat")
+  when: seapath_distro == "CentOS"
```

Comment:
import_playbook does not take "when" into account

Reply:
yes, sorry, I wasn't clear but that's why I meant

Reply:
What would you propose instead ?
`import_playbook` cannot be used inside a play, so not possible to surround it with a block or another statement on which to put the `when`.
The other way would be to put multiple `when` directly in the playbooks be we would then still get multiple "skipping" messages, and it would make a lot of repetitive `when` conditionals

Reply:
I would probably go with having separate "main" playbooks for each flavor, or leave it to the user to create its own playbook.
But this is future work and is not related to this PR.